exporter/awsxrayexporter/translator/segment.go

```diff
@@ -274,10 +283,9 @@ func convertToAmazonTraceID(traceID pdata.TraceID) string {
 	// the traceID, then the epoch may be outside the accepted AWS range of within the
 	// past 30 days.
 	//
-	// In that case, we use the current time as the epoch and accept that a new span
-	// may be created
+	// In that case, we return invalid traceid error
 	if delta := epochNow - epoch; delta > maxAge || delta < -maxSkew {
-		epoch = epochNow
+		return "", errors.New("invalid xray traceid: " + traceID.HexString())
```

Comment:
Think fmt.Errorf is more conventional for creating an error from a formatted string.

Reply:
Thx for pointing out! Let me make the change